dali/operators/reader/loader/webdataset/tar_utils_test.cc

```diff
@@ -121,15 +121,16 @@ TEST_P(SimpleTarTests, Contents) {
   TAR* handle;
   ASSERT_EQ(tar_open(&handle, GetParam().filepath.c_str(), NULL, O_RDONLY, 0, TAR_GNU), 0);
 
-  do {
-    th_read(handle);
+  while(!archive.EndOfArchive()) {
+    ASSERT_EQ(th_read(handle), 0);
```

Comment:
Would we enter here in case of empty archive?

Reply:
nope